docs/_config.yml

```diff
@@ -0,0 +1,49 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: Dymos
+author: The Dymos Development Team
+logo: dymos_logo.png
+exclude_patterns : [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints", "template.ipynb"]
+
+# Force re-execution of notebooks on each build.
+# See https://jupyterbook.org/content/execute.html
+execute:
+  execute_notebooks: force
+  stderr_output: show
+  allow_errors: false
+
+# Define the name of the latex output file for PDF builds
+latex:
+  latex_documents:
+    targetname: book.tex
+
+# Add a bibtex file so that we can create citations
+bibtex_bibfiles:
+  - references.bib
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/OpenMDAO/dymos  # Online location of your book
+  path_to_book: docs  # Optional path to your book, relative to the repository root
+  branch: master  # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+
+# Enable use of latex math
+parse:
+  myst_enable_extensions:
+    # don't forget to list any other extensions you want enabled,
+    # including those that are enabled by default!
+    - amsmath
+    - colon_fence
+    - dollarmath
+    - linkify
+    - substitution
+
+launch_buttons:
+  colab_url: "https://colab.research.google.com"
```

Comment:
I think you'll need this section too if you are embedding Sphinx directives (don't know if the dymos docs do this though) for docstrings.

```# Enable use of sphinx autodoc
sphinx:
  extra_extensions:
  - 'sphinx.ext.autodoc'
  - 'sphinx.ext.autosummary'
  - 'sphinx.ext.viewcode'
  - 'numpydoc'
```

Reply:
Yeah I left those out because it isn't using sphinx directives